min_and_max.py

```diff
@@ -0,0 +1,16 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
+list_without_words = []
+
+
+def min_and_max_value(list_numbers):
+    for item in list_numbers:
+        try:
+            list_without_words.append(int(item))
+        except:

```

Comment:
When the Exception type is not specified, all build-in exceptions will be used including system-exiting exceptions. It means _except:_ will handle Ctrl + C command, for example. I think this behavior is not expected here. 
In my previous comment, I asked you to define and use concrete exceptions that can occur during this function execution.  Please for more information see - https://docs.python.org/3/library/exceptions.html#concrete-exceptions
Also, feel free to reach out to me in case of questions